include/fat.h

```diff
@@ -39,88 +39,90 @@ enum fat_cache_type {
     FAT_CACHE_MAX
 };
 
+// clang-format off
 typedef struct {
-    _Bool       valid;                  /* 0x000 */
-    _Bool       dirty;                  /* 0x001 */
-    uint32_t    max_lba;                /* 0x004 */
-    uint32_t    load_lba;               /* 0x008 */
-    uint32_t    prep_lba;               /* 0x00C */
-    int         sector_cnt;             /* 0x010 */
+    /* 0x000 */ _Bool valid;
+    /* 0x001 */ _Bool dirty;
+    /* 0x004 */ uint32_t max_lba;
+    /* 0x008 */ uint32_t load_lba;
+    /* 0x00C */ uint32_t prep_lba;
+    /* 0x010 */ int sector_cnt;
     _Alignas(0x10)
```

Comment:
this should be attached to `char data[0x800]` instead of floating out in space (although I'm not sure what it's doing here anyway)